docs/source/pages/community/guiding-principles.md

```diff
@@ -0,0 +1,63 @@
+# Guiding Principles
+
+In this document we lay out our vision for
+[**NeuroBlueprint**](https://neuroblueprint.neuroinformatics.dev/)
+and **datashuttle** through three guiding principles.
+
+For additional detail, please refer to our blog posts motivating
+[**NeuroBlueprint**](https://neuroinformatics.dev/blog/neuroblueprint.html)
+and
+[**datashuttle**](https://neuroinformatics.dev/blog/datashuttle.html).
+
+The three guiding principles driving the development of
+**NeuroBlueprint**
+and **datashuttle** are:
+
+## Align with existing community initiatives
+
+[BIDS](https://bids.neuroimaging.io/)
+and
+[NWB](https://www.nwb.org/)
+are the gold-standard for standardising systems neuroscience projects.
+Alignment with these specifications should always be the ultimate goal for any project.
+
+**NeuroBlueprint**'s role is to provide a lightweight standard that can be used to get
+started, especially during the early phase of a project when things may be very busy.
+The guiding principle is that some standardisation is better than no standardisation.
+
+**NeuroBlueprint** is not intended a rival or replacement for more established
+specifications, but rather as a stepping stone towards full standardisation.
+We aim to align as closely as possible to the BIDS specification and in future
+to provide conversion to NWB through **datashuttle**.
+
+## Strive to be lightweight
+
+**NeuroBlueprint** aims to keep the specification as lightweight
+as possible. There is no benefit in the specification proliferating
+as it develops such that it ends up duplicating BIDS in scope.
+
+In the initial versions (**datashuttle** v0.4, **NeuroBlueprint** v0.2),
+the goal is to have a simple organisational
+system in which the raw data for different datatypes (ephys, behaviour)
+can be automatically discovered in any given project.
+
+In future versions it will be necessary to standardise additional features
+(e.g. sync pulse metadata) to allow automation of more sophisticated analyses.
+New standards will always be goal-orientated and customisable, only required
+to achieve a particular research goal. It is anticipated that as these 'rules'
+are accumulated, projects will progressively approach full BIDS standardization.
```

Comment:
Is it worth explaining this more? It might be confusing to readers that neuroblueprint is both:
- Simpler than BIDS
- Will end up being the same as BIDS

I.e. does this mean its simpler now, but won't be in the future?

Reply:
Yes very true, in the end I removed it because the point I am trying to get at is too long to explain / slightly vague.

Basically, the idea was you might use NB for folder structure. Then you want sync pulse standardisation so you use the metadata format and add a key in for sync pulses. You are still not full BIDS because they have a lot of required metadata keys that we don't care about, and your not NWB. But one day you decide to go full BIDS. Now all you need to do is add in these metadata keys and use a datashuttle converter to convert to NWB. So what is required by NB is only a small subset of BIDS but because you took a few steps in this direction, going to full BIDS is much easier.

Reply:
Great thanks thats much better, added